docs/notebooks/02-optimal-planning.ipynb

```diff
@@ -17,31 +21,33 @@
     "id": "t8dCcpf7mivV"
    },
    "source": [
-    "## Setup the library\n",
+    "### Setup\n",
     "\n",
     "We install (from github) the unified planning library."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "id": "lk0GFcWndS8c"
-   },
-   "outputs": [],
-   "source": [
-    "# begin of installation"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 11,
    "metadata": {
     "id": "BoqALxJWdfl8"
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: unified-planning[fast-downward] in /home/shasthamsa/work/drone-experiment/genom3-experiment/env/lib/python3.8/site-packages (0.5.0.216.dev1)\n",
+      "Requirement already satisfied: networkx in /home/shasthamsa/work/drone-experiment/genom3-experiment/env/lib/python3.8/site-packages (from unified-planning[fast-downward]) (2.8.8)\n",
+      "Requirement already satisfied: pyparsing in /home/shasthamsa/work/drone-experiment/genom3-experiment/env/lib/python3.8/site-packages (from unified-planning[fast-downward]) (3.0.9)\n",
+      "Requirement already satisfied: up-fast-downward==0.1.2 in /home/shasthamsa/work/drone-experiment/genom3-experiment/env/lib/python3.8/site-packages (from unified-planning[fast-downward]) (0.1.2)\n"
+     ]
+    }
+   ],
```

Comment:
Do we want to have outputs in the notebooks?

Reply:
It was easier to reproduce the documentation if we stored the output in the notebooks. Different notebook kernels produce unexpected behaviours when the documentation is deployed or served locally.